pkg/crc/machine/bundle/repository_test.go

```diff
@@ -33,7 +33,7 @@ func TestUse(t *testing.T) {
 	assert.NoError(t, err)
 	assert.Equal(t, "4.6.1", bundle.ClusterInfo.OpenShiftVersion)
 
-	bin, err := ioutil.ReadFile(filepath.Join(ocBinDir, constants.OcExecutableName))
+	bin, err := ioutil.ReadFile(filepath.Join(ocBinDir, "oc"))
```

Comment:
looks like this is what failing the windows unit test where `oc.exe` should be the file name so `constants.OcExecutableName` , I think this commit you added only for testing purpose?

Reply:
Yup, the fixup commit is only for testing for now, it will be gone (squashed/removed) when appveyor is fixed. it's using the crc_libvirt_4.6.1 bundle, so this will contain an 'oc' file, but this will get copied/symlinked as oc.exe. 
But it's failing before that line, `Error Trace:	repository_test.go:33` is `assert.NoError(t, err)`

Reply:
Phew, appveyor finally passed 🎉